tests/Integration/Core/Test_ElasticsearchDB.py

```diff
@@ -16,468 +14,490 @@
+#               'bool': {
+#                   'filter': [
+#                       {
+#                           'bool': {
+#                               'must': [
+#                                   {
+#                                       'range': {
+#                                           'timestamp': {
+#                                               'gte': 1423399451544, 'lte': 1423631917911}}}]}}]}}, 'aggs': {
+#               '2': {
+#                   'terms': {
+#                       'field': 'Product', 'size': 0}, 'aggs': {
+#                       'end_data': {
+#                           'date_histogram': {
+#                               'field': 'timestamp', 'interval': '3600000ms'}, 'aggs': {
+#                               'tt': {
+#                                   'terms': {
+#                                       'field': 'timestamp'}, 'aggs': {
+#                                       'total_quantity': {
+#                                           'sum': {
+#                                               'field': 'quantity'}}}}, 'avg_buckets': {
+#                                   'avg_bucket': {
+#                                       'buckets_path': 'tt>total_quantity', 'gap_policy': 'insert_zeros'}}}}}}}})
+#   result = s.execute()
+#   assertEqual(len(result.aggregations['2'].buckets), 2)
+#   assertEqual(result.aggregations['2'].buckets[0].key, u'a')
+#   assertEqual(result.aggregations['2'].buckets[1].key, u'b')
+#   assertEqual(result.aggregations['2'].buckets[0]['end_data'].buckets[0].avg_buckets, {u'value': 2.5})
+#   assertEqual(result.aggregations['2'].buckets[1]['end_data'].buckets[0].avg_buckets, {u'value': 4})
+@pytest.fixture
+def setUpAndTearDown():
+    elasticSearchDB = ElasticSearchDB(host=elHost, port=elPort, useSSL=False)
+    result = elasticSearchDB.createIndex("my-index", {})
+    assert result["OK"]
+    result = elasticSearchDB.index(
+        indexName="my-index", body={"quantity": 1, "Product": "a", "timestamp": 1458226213000}, docID=1
+    )
+    assert result["OK"]
+
+    yield setUpAndTearDown
+
+    elasticSearchDB.deleteIndex("my-index")
+
+
+def test_getAndExistsDoc(setUpAndTearDown):
+    res = elasticSearchDB.existsDoc(index="my-index", id=1)
+    assert res
+    res = elasticSearchDB.getDoc(index="my-index", id=1)
+    assert res["Value"] == {"quantity": 1, "Product": "a", "timestamp": 1458226213000}
+
+
+def test_updateDoc(setUpAndTearDown):
+    res = elasticSearchDB.updateDoc(index="my-index", id=1, body={"doc": {"quantity": 2, "Product": "b"}})
+    print(res)
+    assert res["OK"]
+    res = elasticSearchDB.getDoc(index="my-index", id=1)
+    assert res["Value"] == {"quantity": 2, "Product": "b", "timestamp": 1458226213000}
+
+
+def test_deleteDoc(setUpAndTearDown):
+    res = elasticSearchDB.deleteDoc(index="my-index", id=1)
+    assert res["OK"]
+    res = elasticSearchDB.getDoc(index="my-index", id=1)
+    assert res["OK"], res["Value"] == {}
```

Comment:
I am not sure what this last assert is supposed to do.
It tests that `res["OK"]` is true, and if not, it will print the eval of `res["Value"] == {}`
I have the feeling you rather want 

```python
assert res["OK"]
assert res["Value"] == {}
```